src/components/TablePreview/TablePreview.tsx

```diff
@@ -32,7 +31,11 @@ const TablePreview: FC<Props> = ({}) => {
             <table ref={tableRef}>
                 <tbody>
                     <tr>
-                        <td rowSpan={3}>Image Taking 3 cells</td>
+                        <td rowSpan={3}>
+                            <Flex alignItems="center">
+                                <img src={imgUrl} height="60"></img>
+                            </Flex>
```

Comment:
Uff solid approach! Pero **_plot-twist_**!  Recuerde que el table para el signature tiene que ser `pure html` con inline styling (no react components allowed) 

Metamosle un comment en la linea 7 para que no se nos olvide esto

```javascript
/**
* This component *must* be pure html with inline styling, otherwise we will not be able to copy & paste it into an email.
*/
```